src/Microsoft.AspNetCore.Server.IISIntegration.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.AspNetCore.Server.IISIntegration</id>
+    <version>$version$</version>
+    <description>ASP.NET Core components for working with the IIS AspNetCoreModule.</description>
+    <tags>aspnetcore,iis</tags>
+    <serviceable>true</serviceable>
+    <dependencies>
+      <group targetFramework=".NETFramework4.5.1">
+        <dependency id="Microsoft.AspNetCore.Hosting.Abstractions" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.Http" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.Http.Extensions" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.HttpOverrides" version="$version$" />
+        <dependency id="Microsoft.Extensions.Logging.Abstractions" version="$version$" />
+        <dependency id="Microsoft.Extensions.Options" version="$version$" />
+      </group>
+      <group targetFramework=".NETStandard1.3">
+        <dependency id="System.Security.Principal.Windows" version="4.0.0" />
+        <dependency id="Microsoft.AspNetCore.Hosting.Abstractions" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.Http" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.Http.Extensions" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.HttpOverrides" version="$version$" />
+        <dependency id="Microsoft.Extensions.Logging.Abstractions" version="$version$" />
+        <dependency id="Microsoft.Extensions.Options" version="$version$" />
```

Comment:
Oh yeah, we aren't guaranteed that anymore (cough buildtrain)...

Btw, this nuspec isn't actually in use yet. We're still waiting on MSBuild to figure out how to distribute/build tasks in a nupkg. (ping @davidfowl )